03-transactions.md

```diff
@@ -484,6 +486,112 @@ A node:
   - if the resulting fee rate is too low:
     - MAY fail the channel.
 
+## Dust Limits
+
+The `dust_limit_satoshis` parameter is used to configure the threshold below
+which nodes will not produce on-chain transaction outputs.
+
+There is no consensus rule in Bitcoin that makes outputs below dust thresholds
+invalid or unspendable, but policy rules in popular implementations will prevent
+relaying transactions that contain such outputs.
+
+Bitcoin Core defines the following dust thresholds:
+
+- pay to pubkey hash (p2pkh): 546 satoshis
+- pay to script hash (p2sh): 540 satoshis
+- pay to witness pubkey hash (p2wpkh): 294 satoshis
+- pay to witness script hash (p2wsh): 330 satoshis
+- unknown segwit versions: 354 satoshis
+
+The rationale of this calculation (implemented [here](https://github.com/bitcoin/bitcoin/blob/0.21/src/policy/policy.cpp))
+is explained in the following sections.
+
+In all these sections, the calculations are done with a feerate of 3000 sat/kB
+as per Bitcoin Core's implementation.
+
+### Pay to pubkey hash (p2pkh)
+
+A p2pkh output is 34 bytes:
+
+- 8 bytes for the output amount
+- 1 byte for the script length
+- 25 bytes for the script (`OP_DUP` `OP_HASH160` `20` 20-bytes `OP_EQUALVERIFY` `OP_CHECKSIG`)
+
+A p2pkh input is at least 148 bytes:
+
+- 36 bytes for the previous output (32 bytes hash + 4 bytes index)
+- 1 byte for the script sig length
```

Comment:
nit: switch this line with below so script sig length is right before script sig?

Reply:
Done in https://github.com/lightningnetwork/lightning-rfc/pull/894/commits/dff76b23e81a42184e861d7d7b996a81e578f206